db/migrate/20220806012249_add_order_to_columns.rb

```diff
@@ -0,0 +1,15 @@
+class AddOrderToColumns < ActiveRecord::Migration[7.0]
+  class Column20220806 < ActiveRecord::Base
+    self.table_name = :columns
+  end
+
+  def change
+    change_table :columns do |t|
+      t.integer :column_order
```

Comment:
Should column_order be nullable?

Reply:
Great question @molenick-mov! The ranked-model gem requires that its order column be non-nullable.